README.rst

```diff
@@ -43,7 +43,7 @@ Development Workflow
 --------------------
 
 One Time Setup
-______________
+~~~~~~~~~~~~~~
```

Comment:
**Note to reviewer**: [Mark resolved once seen.] These need to match the change in CHANGELOG, because these two files are joined during doc build.